app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -211,39 +211,20 @@ class SupplementalApplicationPage extends React.Component {
     }
   }
 
-  handleSaveNewRentalAssistance = async (rentalAssistance) => {
-    // Salesforce does not accept currency strings. TODO: Move this into the form
+  handleSaveRentalAssistance = async (rentalAssistance, action = 'create') => {
```

Comment:
I consolidated `handleSaveNewRentalAssistance` and `handleUpdateRentalAssistance` into `handleSaveRentalAssistance` that takes an action type because the code is so similar.

I kept `rentalAssistance['assistance_amount'] = currencyToFloat(rentalAssistance['assistance_amount'])` here because doing this in the form means putting it in two different places (for the update in the Panel, and the create in the form) on the submit function. It didn't make sense to embed it in the input because we want the input to display the currency format.

Reply:
Moved to rails side in 897bb41